commands/zrange.md

```diff
@@ -17,23 +17,23 @@ The optional `WITHSCORES` argument supplements the command's reply with the scor
 
 ## Index ranges
 
-By default, the command performs an index range query. The `<min>` and `<max>` arguments represent zero-based indexes, where `0` is the first element, `1` is the next element, and so on. These arguments specify an **inclusive range**, so for example, `ZRANGE myzset 0 1` will return both the first and the second element of the sorted set.
+By default, the command performs an index range query. The `<start>` and `<stop>` arguments represent zero-based indexes, where `0` is the first element, `1` is the next element, and so on. These arguments specify an **inclusive range**, so for example, `ZRANGE myzset 0 1` will return both the first and the second element of the sorted set.
 
 The indexes can also be negative numbers indicating offsets from the end of the sorted set, with `-1` being the last element of the sorted set, `-2` the penultimate element, and so on.
 
 Out of range indexes do not produce an error.
 
-If `<min>` is greater than either the end index of the sorted set or `<max>`, an empty list is returned.
+If `<start>` is greater than either the end index of the sorted set or `<stop>`, an empty list is returned.
 
-If `<max>` is greater than the end index of the sorted set, Redis will use the last element of the sorted set.
+If `<stop>` is greater than the end index of the sorted set, Redis will use the last element of the sorted set.
```

Comment:
I think we should update this explanation so it is correct also with REV.

Reply:
These statements still appear to be accurate, if `BYSCORE` is not used it still min/max would still apply in the same positions, it just reverses the result. Might warrant some further discussion about consistency - I will write up something for the `REV` argument with some example usage.

Reply:
Oh, good. It's confusing indeed, but your last commit explains it well.